coreneuron/CMakeLists.txt

```diff
@@ -34,6 +34,10 @@ file(GLOB_RECURSE coreneuron_all_templates "*.ipp")
 file(GLOB_RECURSE coreneuron_cuda_files "*.cu")
 file(GLOB_RECURSE coreneuron_all_c_files "nrniv/*.c*" "nrnmpi/*.c*" "nrnoc/*.c*"
                                          "nrnomp/*.c*" "utils/*.c*")
+if(NOT ENABLE_ISPC_INTEROP)
+    list(FILTER coreneuron_all_c_files EXCLUDE REGEX .*/utils/ispc_globals.c)
+endif()
```

Comment:
can we change `ispc_globals.c` to `ispc_globals.cpp` with c linkage? so that later we wont have confusion about c vs cpp codebase. (this will be a single C file in the project)